### ISSUE TITLE
Burrowing does not affect redstone ore

### ISSUE THREAD

**Issue by Boingboingsplat** [Azanor/thaumcraft-5#476]
A pickaxe with burrowing mines redstone ore as if it's a normal block. I suspect this is related to the fact that redstone ore changes data values to glow once it's touched.

**Comment by Azanor** [comment]
Yup that would be it. Hmm, I'll have to take a look and see if I can do anything about it.

**Comment by Azanor** [comment]
Messed around for a few hours and there is no simple or generic way to handle redstone ore and hardcoding specifically for it is very messy - the changing blockstate causes havoc with my burrow code. I'll keep this open and try again in the future, but for now I'm labeling this as won't fix.
